cmd/release/cmd/tag.go

```diff
@@ -279,6 +280,49 @@ var dashboardTagSubCmd = &cobra.Command{
 	},
 }
 
+var cliTagSubCmd = &cobra.Command{
+	Use:   "cli [ga,rc,alpha,test] [version]",
+	Short: "Tag dashboard releases",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 2 {
+			return errors.New("expected at least two arguments: [ga,rc,alpha,etc] [version]")
```

Comment:
Can you double check the notes to make sure the CLI gets these meta release types?

Reply:
I checked my meeting notes and I pointed out the need for `alpha`, `rc`, `test` and `ga`, but going through the `rancher/cli` tags I only see `ga` and `rc` (also `alpha` but those are really old version `<= v2.3.0`). I'll check with CLI engineers.

Reply:
Confirmed with CLI engineers, they only need `ga` and `rc` releases.